src/vitabel/timeseries.py

```diff
@@ -1890,32 +1978,55 @@ def plot(
         if plotstyle is not None:
             base_plotstyle.update(plotstyle)
 
-        if data is not None:
-            # TODO: deal with data string entries (or disallow them)
-            if "fmt" not in base_plotstyle:
-                base_plotstyle["fmt"] = "none"
-            if "capsize" not in base_plotstyle:
-                base_plotstyle["capsize"] = 3
-            artist = plot_axes.errorbar(
-                time_midpoints, data, xerr=time_radius, **base_plotstyle
-            )
-        else:
+        if plot_type in {'box', 'combined'}:
             if "alpha" not in base_plotstyle:
-                base_plotstyle["alpha"] = 0.2
+                base_plotstyle.update({"alpha": 0.2})
             if "color" not in base_plotstyle:
-                base_plotstyle["color"] = "blue"
+                base_plotstyle.update({"color" :"blue"})
```

Comment:
any particular reason you changed the direct assignments to updates?

```suggestion
                base_plotstyle.update({"color": "blue"})
```